R/notin.R

```diff
@@ -0,0 +1,7 @@
+"%notin%" = function(example, elements) {
+  if (is.character(example)) {
+    return(.Call(Cchin, example, elements, TRUE))
```

Comment:
this branch doesn't have any tests:

https://app.codecov.io/gh/Rdatatable/data.table/compare/4931/diff#D1L4

Reply:
Looking at the tests it should be covered. Maybe it hasn't re-run coverage yet.

Reply:
That link doesn't work for me, but I do have a couple of tests.

Reply:
seems to have gone away on merge thanks